src/screens/NearbyLocations.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, TextInput, FlatList, TouchableOpacity } from 'react-native';
+import firebase from 'firebase/app';
+import 'firebase/database';
+
+
+const App = () => {
```

Comment:
Shouldn't be called App...
Each React Native App only should have one file called App.js
This should have been renamed to NearbyLocation...